src/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      data-cy="Nav"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            to="/tabs"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Tabs
+          </NavLink>
```

Comment:
I'd recommend to create an array of objects for NavLink's, than map this array inside <div className="navbar-brand">